packages/gatsby-plugin-google-gtag/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="1.0.0-beta.0"></a>
+
+# 1.0.0-beta.0 (2018-06-17)
+
+**Note:** Version bump only for package gatsby-plugin-google-gtag
```

Comment:
Where did that changelog came from?

Reply:
Forgot about that, I copied it from one of the other plugins. Should I just update the date?

Reply:
@tylerbuchea let's just go ahead and remove it if that's OK! lerna should handle the generation for us when we're ready to release!

Reply:
Got you, yep sounds good 👍